week-1-practise/level-2/2-format-currency.js

```diff
@@ -6,7 +6,10 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency(currency) {
+  var result = "£" + currency.toFixed(2);
```

Comment:
Nice! Do you think you can rewrite this function, so that you don't need the variable at all?

Reply:
@martynamaron, yes I can rewrite the function without the variable I.e    function formatCurrency(currency) {
  return "£" + currency.toFixed(2);
}